lib/AST/ASTImporter.cpp

```diff
@@ -2914,13 +2917,13 @@ Decl *ASTNodeImporter::VisitFriendDecl(FriendDecl *D) {
                                        ToTPLists);
 
   Importer.Imported(D, FrD);
+  RD->pushFriendDecl(FrD);
```

Comment:
This line is present in the code at https://reviews.llvm.org.

Reply:
Will be removed because causes this test to "hang":
libclang/./libclangTests/libclang.clang_createTranslationUnit_InvalidArgs

Reply:
`pushFriendDecl` creates the list for all the friends of a specific class. Actually we should have a test to check if the list content is correct after `Import()`.
Thus, "hang" is bad. Could you please further investigate? Perhaps in another PR.

Reply:
`pushFriendDecl` is called by `FriendDecl::Create`, no need to call it again. If called again it causes some infinite loop when the friend list is iterated.

Reply:
OK.